### ISSUE TITLE
Attribute android:textIsSelectable="true" is not working

### ISSUE THREAD

**Issue by BlueMango92** [SufficientlySecure/html-textview#59]
`<org.sufficientlysecure.htmltextview.HtmlTextView`
           `android:layout_width="match_parent"`
            `android:layout_height="wrap_content"`
            `android:textSize="16sp"`
            `android:textIsSelectable="true"/>`

**Comment by dschuermann** [comment]
``textIsSelectable`` seems to be extremely buggy for TextViews in general (https://www.google.de/search?hl=de&q=textIsSelectable).

**Comment by BlueMango92** [comment]
I never had a problem with the standard TextViews. Is there a workaround to make it works? I just need to be able to copy the selected text.
By the way thanks for the answer.

**Comment by ofalvai** [comment]
In my case the exact problem was that on Marshmallow and above, the text could be selected, but the text selection commands didn't appear above the textview.
I think the root cause is the overridden `HtmlTextView.onTouchEvent()` method, and that the `mDontConsumeNonUrlClicks` field set to true, because currently the method always returns false thanks to that boolean. If I set `mDontConsumeNonUrlClicks` to true, the text selection bubble appears above the text.

Currently there's no way to control that field outside the package (I modified the code of `HtmlTextView` to test my solution), but I would gladly make a pull request with a setter method. I'm just not sure what's the purpose of `mDontConsumeNonUrlClicks` currently, and whether it could be removed altogether or a setter would be better.
